inc/cropper/src/views/image-edit.js

```diff
@@ -15,12 +16,43 @@ const ImageEditView = Media.View.extend( {
 			this.model.set( { size: 'full' } );
 		}
 
+		// Get current size from block attributes if available.
+		this.setSizeFromBlock();
+
 		// Re-render on certain updates.
 		this.listenTo( this.model, 'change:url', this.onUpdate );
 
 		// Initial render.
 		this.onUpdate();
 	},
+	setSizeFromBlock() {
+		if ( ! wp || ! wp.data ) {
+			return;
+		}
+
+		const selectedBlock = wp.data.select( 'core/block-editor' ).getSelectedBlock();
+		if ( ! selectedBlock ) {
+			return;
+		}
+
+		const sizeForBlock = applyFilters(
+			`smartmedia.cropper.selectSizeFromBlockAttributes.${ selectedBlock.name.replace( /\W+/g, '.' ) }`,
+			null,
+			selectedBlock
+		);
+
+		const size = applyFilters(
+			`smartmedia.cropper.selectSizeFromBlockAttributes`,
```

Comment:
Looks like no need for templated string here.

Reply:
ah! cheers, left over copy/pasta